lib/msf/core/post/file.rb

```diff
@@ -501,19 +501,25 @@ def rm_rf(*remote_dirs)
   alias :dir_rm :rm_rf
 
   #
-  # Rename a remote file.
+  # Renames a remote file and returns true on success and false
+  # on failure
   #
   # @param old_file [String] Remote file name to move
   # @param new_file [String] The new name for the remote file
   def rename_file(old_file, new_file)
+    unless exist?(old_file)
+      elog("The file to be renamed does not exists!")
+      return false
+    end
+    verification_token = Rex::Text.rand_text_alphanumeric()
```

Comment:
`#rand_text_alphanumeric` needs a length parameter.
```
>> verification_token = Rex::Text.rand_text_alphanumeric()
Traceback (most recent call last):
	27: from ./msfconsole:23:in `<main>'
	26: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/metasploit/framework/command/base.rb:82:in `start'
	25: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/metasploit/framework/command/console.rb:48:in `start'
	24: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/shell.rb:155:in `run'
	23: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/dispatcher_shell.rb:475:in `run_single'
	22: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/dispatcher_shell.rb:475:in `each'
	21: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/dispatcher_shell.rb:481:in `block in run_single'
	19: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/msf/ui/console/command_dispatcher/developer.rb:113:in `cmd_irb'
	18: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/shell/history_manager.rb:44:in `with_context'
	17: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/msf/ui/console/command_dispatcher/developer.rb:117:in `block in cmd_irb'
	16: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/irb_shell.rb:52:in `run'
	15: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/irb_shell.rb:52:in `catch'
	14: from /home/smcintyre/Repositories/metasploit-framework.pr/lib/rex/ui/text/irb_shell.rb:53:in `block in run'
	 1: from (irb):18:in `<main>'
/home/smcintyre/.rvm/gems/ruby-2.7.2@metasploit-framework/gems/rex-text-0.2.34/lib/rex/text/rand.rb:136:in `rand_text_alphanumeric': wrong number of arguments (given 0, expected 1..2) (ArgumentError)
```

Reply:
Updated this :+1: